runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -21,6 +21,7 @@
 import org.corfudb.runtime.exceptions.OutrankedException;
 import org.corfudb.runtime.exceptions.QuorumUnreachableException;
 import org.corfudb.runtime.exceptions.RecoveryException;
+import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

Reply:
done.